Makefile

```diff
@@ -39,6 +39,11 @@ generate-crds: controller-gen
 # code generation for custom resources
 kubegen:
 	./hack/update-codegen.sh
+	@cp -rf v2/pkg/apis pkg/
+	@cp -rf v2/pkg/client pkg/
+	@cp -rf v2/pkg/internalapis/apis/cstor/v1/zz_generated.conversion.go pkg/internalapis/apis/cstor/v1/zz_generated.conversion.go
+	@cp -rf v2/pkg/internalapis/apis/cstor/zz_generated.deepcopy.go pkg/internalapis/apis/cstor/zz_generated.deepcopy.go
+	@rm -rf v2
```

Comment:
question: Is this because the codegen generates the v2 directory ??

Reply:
Yes, all the generated `clientset, informers, deepcopy` paths has to contain the module path `github.com/openebs/api/v2/...`  so we have to use the same module name in code-gen as well.